src/app/components/core/notifications/notification-modal/notification-modal.component.ts

```diff
@@ -0,0 +1,202 @@
+import { Component, OnInit, Input, Output, EventEmitter, OnChanges, ViewChild } from '@angular/core';
+
+import { isEmpty } from 'lodash';
+
+import {
+  ConfigurationService, AlertService,
+  ProgressBarService,
+  NotificationsService
+} from '../../../../services';
+import { AlertDialogComponent } from '../../../common/alert-dialog/alert-dialog.component';
+import { ViewConfigItemComponent } from '../../configuration-manager/view-config-item/view-config-item.component';
+
+@Component({
+  selector: 'app-notification-modal',
+  templateUrl: './notification-modal.component.html',
+  styleUrls: ['./notification-modal.component.css']
+})
+export class NotificationModalComponent implements OnInit, OnChanges {
+
+  @Input() notification: { notification: any };
+  @Output() notify: EventEmitter<any> = new EventEmitter<any>();
+
+  public useRuleProxy: 'true';
+  public useDeliveryProxy: 'true';
+  public useProxy: 'true';
+  public category: any;
+  public ruleConfiguration: any;
+  public deliveryConfiguration: any;
+  public notificationRecord: any;
+  public changedChildConfig = [];
+
+  rulePluginChangedConfig = [];
+  deliveryPluginChangedConfig = [];
+  notificationChangedConfig = [];
```

Comment:
on modal, even if no config is changed and Save is clicked. old request is being repeated.